pymc4/random_variable.py

```diff
@@ -0,0 +1,23 @@
+from tensorflow_probability import edward2 as ed
+from pymc4 import Model
+
+
+class RandomVariable(ed.RandomVariable):
+
+    def __init__(
+                self,
+                distribution,
+                sample_shape=(),
+                value=None,
+                name="RV"
```

Comment:
the name should probably be the first argument, and have no default argument, right?